frontend/src/components/SendFileFormTest/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Button, Typography } from '@mui/material';
+import React, { useState } from 'react';
+
+function SendFileFormTest() {
```

Comment:
I think you should use the arrow function here

Reply:
This is airbnb code-style, so right now, we cannot use arrow functions, only function declaration
![image](https://user-images.githubusercontent.com/41287687/163373825-a6a2d8bb-cde8-4afb-a703-d10084243f90.png)

Reply:
And when arrow function, error appears (Function component is not a function declarationeslint[react/function-component-definition](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/function-component-definition.md)
![image](https://user-images.githubusercontent.com/41287687/163374248-2a8b7e51-a383-4b23-b239-ec8789c3c850.png)

Reply:
Ok, didn't know about that